apps/portal/examples/react-template/content/source/Dockerfile

```
@@ -20,7 +20,25 @@ COPY --from=builder /app/dist /usr/share/nginx/html
 # Configure port dynamically at runtime/build-time
 ARG PORT=${{ values.port }}
 ENV PORT=${PORT}
-RUN sed -i "s/listen\(  *\)80;/listen ${PORT};/" /etc/nginx/conf.d/default.conf
+
+RUN echo 'server { \
```

Hadolint (2.14.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)